### DISCUSSION TITLE
🌹 ROSE Bloom Grants • Build any project related to the growth and development of the Oasis Network

### DISCUSSION THREAD

**Discussion by jcueto** [oasisprotocol/community-forum#12] [Grants Program]
![image](https://user-images.githubusercontent.com/5066732/106105783-6276ad00-60f9-11eb-8a08-6b07d598e61e.png)
-----
### Apply here: https://oasisfoundation.typeform.com/to/HtYql2aN
-----

_Note: This discussion thread serves as a hub for grant applications related to this project category. You'll see project proposals that have been submitted and reviewed by the Oasis Foundation team. You'll see proposals that have been marked as "ACCEPTED" and "TURNED DOWN," which will give you a better sense of what we're looking for in a grant application. Community members are encouraged to upvote projects, provide feedback, ask questions, and offer suggestions on each project shared on this thread._ 

-----

The ROSE Bloom Grants Program is an entirely new grants program designed to accelerate the growth of the Oasis Network and build valuable new ecosystem tools. Apply for a grant today to bring your project to life!

For this project category, we’re looking for developers who want to pitch us a new project that contributes to the growth and development of the Oasis Network. If you have a project idea and need funds to build it, we've got your back! This is a very open-ended call for proposals. We're open to funding new DApps, DeFi protocols, Oasis blockchain analytics tools, web wallets, web staking/delegating UIs, and anything else you want to work on, as long as it somehow contributes to the Oasis Network. You can use any backend and frontend frameworks you want, and any coding languages you want. We're excited to see what you come up with!

-----

### HOW TO APPLY

Applying for a grant is easy. Just fill out the [application form](https://oasisfoundation.typeform.com/to/HtYql2aN) with the relevant information we are requesting, and we’ll notify you via email regarding the status of your application. 

The more specific and clear you are in your application, the better our team will be able to understand your project proposal.

-----

### GRANT AMOUNT

Each proposal will be eligible to receive anywhere from **$5,000 to $50,000 USD** given out in **ROSE tokens**. Please specify the grant amount you are requesting in your application. The grant amount requested should match the scope of work for each project. 

-----

### PROJECT MILESTONES

We ask that you provide specific project milestones as part of your application. Milestones should be decided based on the type of project you are developing and the scope of work for the project. In general, the types of things you can list as project milestones include specific development steps (e.g. initial deployment, debugging, obtaining first user, etc.), specific components of the application (e.g. sign-in flow, front-end user interface, database config, etc.), as well as key success metrics (e.g. app installs, number of transactions, number of users, etc.).

Depending on the size of your grant, the Oasis Foundation may disburse your grant in separate parts. Smaller grants may be disbursed in a single transaction. Larger grants may be disbursed in separate transactions at separate times, based on evidence of your continued progress building your project and meeting the milestones we agreed upon. 

Please note that we expect a commitment of at least around 3 months, including project development, launch, maintenance, etc. If you want to continue working on your project or developing extensions for it beyond this initial commitment, you can apply for additional grants in the future to fund your continued work. 

-----

### SELECTION CRITERIA

The Oasis Foundation team will assess your application based on the following scoring categories: 
- Strength of the team, based on past projects, resume, GitHub, etc. - 1 (Very Weak) to 5 points (Very Strong) 
- Written quality of project proposal, based on details described, content, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Tech & product quality of project, based on technical implementation details and product vision, design overview, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Relevance to the success of the network, based on whether this is a product that could help us grow, an app that is in-demand among our community members, etc.- 1 (Not very relevant at all) to 5 points (Very Relevant)

Community interest and excitement generated will be a factor considered in the selection process, as an input into the “Relevance to the success of the network” scoring category. We’ll post project overviews on the Oasis GitHub discussion board to allow Oasis community members to share feedback, ask questions, and offer suggestions for each project. 

-----

### APPLICATION TIMELINE 

Grant applications will be reviewed on an ongoing, rolling basis. In general, the application process will follow the timeline below: 
- Apply for a grant via our application form: https://oasisfoundation.typeform.com/to/HtYql2aN 
- Allow 1 to 2 weeks for the Oasis team to do a preliminary review of your application
- If your project idea looks promising, the Oasis team will post your project overview on the Oasis GitHub discussion board and notify you via email (applicants that are turned down at this preliminary stage will also be notified). This will allow the community to share feedback on your project proposal, ask questions, and offer you suggestions.
- Allow 1 to 2 weeks after your project gets posted on the Oasis GitHub discussion board and receives community feedback for the Oasis team to mark your project as ACCEPTED or TURNED DOWN. We will also notify you if your application is approved or turned down at this stage via email.
- If your grant application is accepted, the Oasis team will email you with next steps, including information about the KYC process.
The Oasis team will work closely with you to finalize the terms of your grant, and you can begin working on your project!

-----

### HELPFUL DOCUMENTATION

You can use any backend and frontend frameworks you want for building your project. 

If you're interested in using the Parcel SDK and/or Oasis Ethereum ParaTime, check out the relevant documentation below:
- Parcel SDK documentation: http://docs.oasiscloud.io/ ( use this password: Parcel2020! )
- For access to Parcel SDK please fill out this form: https://oasisfoundation.typeform.com/to/u3EINnMw

Check out the following Parcel SDK tutorials and app demos:
- Parcel in Action: PrivGuard - https://youtu.be/SKYP2Vceag0
- Oasis Harvest Demo - https://youtu.be/9HduApRDJyA
- Parcel in Action: Nebula Genomics - https://youtu.be/VA5la2h75lQ
- Parcel in Action: MusicFund - https://youtu.be/DQqiiLYw1mI

The following documentation should help you leverage the Oasis Ethereum ParaTime:
- Oasis Ethereum ParaTime Website: https://www.oasiseth.org/#developers 
- Oasis Ethereum ParaTime Tutorial - https://youtu.be/NEo7W_qgKos 

For more ideas on the types of projects you can build using Parcel SDK and the Oasis Ethereum ParaTime, you can check out some of the projects that won The Quest for ROSE: An Oasis Network Hackathon: https://medium.com/oasis-protocol-project/announcing-the-winners-of-the-quest-for-rose-hackathon-5d160722a840 

-----

### GET SUPPORT

- Join our Oasis Community Slack to view all of our support channels for developers: https://join.slack.com/t/oasiscommunity/shared_invite/zt-8a71w8i9-sVhe_5IG1hhb5Dgjl5xu2A 
- For logistical questions related to the ROSE Bloom Grants Program, post in the #grants-program-public channel
- For tech questions related to building with Parcel SDK, post in the #hackathon-parcel-support channel
- For tech questions related to building with the Oasis Ethereum ParaTime, post in the #oasiseth-paratime channel
- To learn more about the Oasis Network community of developers and supporters, join our Telegram: https://t.me/oasisprotocolcommunity 

-----

### TERMS & CONDITIONS

Please note that eligibility to get approved for a grant is subject to KYC and your country of residence's accredited investor verification guidelines, as outlined in the official program rules (https://docsend.com/view/xykrayt6vb5tn7eh). 

Developers who are U.S. citizens or residents and who are not accredited investors may not be eligible to receive ROSE Bloom Grants. The grants are not and will not be offered or sold, directly or indirectly, to any person who is a resident, organized, or located in any country or territory subject to OFAC comprehensive sanctions programs from time to time, including Cuba, Crimea region of Ukraine, Democratic People’s Republic of Korea, Iran, Syria, any person found on the OFAC Specially Designated Nationals, Blocked Persons List, any other consolidated prohibited persons list as determined by any applicable governmental authority, or any persons who are organized in, located in, or residents of New York State.

-----

### For other grant project categories, check out our other Grants Program discussion threads: https://github.com/oasisprotocol/community/discussions/categories/grants-program

-----

**Comment by Esya** [comment]
Hi, would a non-custodial pure web/JS wallet that allows to easily stake and check commissions be eligible ? I'm seriously thinking about building this as it seems to be a barrier of entry as the current solutions (Hardware needed with Ledger, Bitpie is mobile only and somewhat unreliable sometimes, and most users are not tech-savy enough to use the CLI) are not super user friendly. 

Let me know your thoughts, if that could potentially be considered, I'll consider submitting a formal application to the grant program.

Thanks !

**Edit :** Sent a formal application regarding this topic, I'll delete this message once it's posted here !
